apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/WalletOverview/WalletOverview.tsx

```diff
@@ -44,23 +47,16 @@ export const WalletOverview: FC<WalletOverviewProps> = ({
       {account ? (
         <>
           <div>
-            {['ETH', 'WETH'].includes(asset) && (
+            {ETH_ASSET_SYMBOLS.includes(asset) && (
               <Tabs
                 className="mb-5"
-                index={asset === 'ETH' ? 0 : 1}
-                onChange={e => setAsset(e === 0 ? 'ETH' : 'WETH')}
-                items={[
-                  {
-                    activeClassName: 'text-primary-20',
-                    dataAttribute: 'eth',
-                    label: 'ETH',
-                  },
-                  {
-                    activeClassName: 'text-primary-20',
-                    dataAttribute: 'weth',
-                    label: 'WETH',
-                  },
-                ]}
+                index={ETH_ASSET_SYMBOLS.indexOf(asset)}
+                onChange={e => setAsset(ETH_ASSET_SYMBOLS[e])}
+                items={ETH_ASSET_SYMBOLS.map(s => ({
```

Comment:
It will recreate the array on every rerender, avoid this if components are complex

Reply:
I think should be fine for now, should be just 2 symbols and just for 1 token so...